src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/__tests__/ContactDiaryStoreV1Tests.swift

```diff
@@ -51,7 +51,7 @@ class ContactDiaryStoreV1Tests: XCTestCase {
 
 		XCTAssertEqual(name, "Hinterm Mond")
 	}
-
+	
```

Comment:
The old implementation of the store can be removed. In case of the packages store, the old implementation was only used for "faster" testing purpose. After discussing about that, its better to remove the old implementation and only use the Schema for migration testing.